web/src/app/schedules/CalendarToolbar.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+import {
+  Button,
+  ButtonGroup,
+  ButtonProps,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core'
+import { DateTime } from 'luxon'
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    paddingBottom: '1em',
+  },
+  labelGridItem: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    order: 3,
+    [theme.breakpoints.up('lg')]: {
+      order: 2,
+    },
+  },
+  primaryNavBtnGroup: {
+    flex: 1,
+    display: 'flex',
+    justifyContent: 'flex-start',
+    order: 1,
+  },
+  secondaryBtnGroup: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    order: 2,
+    [theme.breakpoints.up('lg')]: {
+      justifyContent: 'flex-end',
+      order: 3,
+    },
+  },
+  actionBtn: {
+    marginLeft: theme.spacing(1),
+  },
+}))
+
+type ViewType = 'month' | 'week'
+interface CalendarToolbarProps {
+  date: Date
+  label: string
+  onNavigate: (e: React.MouseEvent, date: Date) => void
+  onView: (view: ViewType) => void
+  view: ViewType
+  actionButtonProps?: ButtonProps
+}
+
+function CalendarToolbar(props: CalendarToolbarProps): JSX.Element {
+  const classes = useStyles()
+  const weekly = props.view === 'week'
+
+  return (
+    <Grid container spacing={2} className={classes.container}>
+      <Grid item xs={12} lg={4} className={classes.primaryNavBtnGroup}>
+        <ButtonGroup color='primary' aria-label='Calendar Navigation'>
+          <Button
+            data-cy='show-today'
+            onClick={(e) => props.onNavigate(e, DateTime.local().toJSDate())}
+          >
+            Today
+          </Button>
+          <Button
+            data-cy='back'
+            onClick={(e) => {
+              const timeUnit = weekly ? { weeks: 1 } : { months: 1 }
+              const nextDate = DateTime.fromJSDate(props.date)
+                .minus(timeUnit)
+                .toJSDate()
+
+              props.onNavigate(e, nextDate)
+            }}
```

Comment:
I'd prefer to keep the action handlers/logic-y bits out of the JSX portion of the component